docker/rootless/etc/templates/app.ini

```diff
@@ -0,0 +1,57 @@
+APP_NAME = $APP_NAME
+RUN_USER = $RUN_USER
+RUN_MODE = $RUN_MODE
+
+[repository]
+ROOT = $GITEA_WORK_DIR/git/repositories
+
+[repository.local]
+LOCAL_COPY_PATH = $GITEA_WORK_DIR/data/tmp/local-repo
+
+[repository.upload]
+TEMP_PATH = $GITEA_WORK_DIR/data/uploads
```

Comment:
During the review of temporary path, I ask myself if this path variable is still used.
I have found references here:
- https://github.com/go-gitea/gitea/blob/171b3598778a1ecd0a921c71ed6755bfef68f7f0/models/upload.go#L38
- https://github.com/go-gitea/gitea/blob/c8d7ae1ee33e31ed15632a26c74ba9a3ea115043/modules/setting/repository.go#L226
But it seems never used anymore.